src/CovidAppointmentTable.js

```diff
@@ -121,29 +134,87 @@ export default function CovidAppointmentTable() {
                     </div>
                 )}
                 <div role="list">
-                    {formattedData.map((entry) => {
-                        return (
-                            <div
-                                role="listitem"
-                                key={`${entry.location}-${entry.streetAdress}-${entry.city}`}
-                                className={classes.cardBox}
-                            >
-                                <Card>
-                                    <CardHeader
-                                        title={<div>{entry.location}</div>}
-                                        subheader={<div>{entry.city}</div>}
-                                    />
-                                    <CardContent>
-                                        <Availability entry={entry} />
-                                        <MoreInformation entry={entry} />
-                                        <SignUpLink entry={entry} />
-                                    </CardContent>
-                                </Card>
-                            </div>
-                        );
-                    })}
+                    {formattedData.map((entry) => (
+                        <LocationCard
+                            entry={entry}
+                            className={classes.cardBox}
+                            key={`${entry.location}-${entry.streetAdress}-${entry.city}`}
+                        />
+                    ))}
                 </div>
             </section>
         </>
     );
 }
+
+function RestrictionNotifier({ entry }) {
+    let hasRestriction = false;
+    let restrictionText = null;
+
+    if (entry.restrictions) {
+        hasRestriction = true;
+        restrictionText = entry.restrictions;
+    } else if (entry.extraData && entry.extraData["Additional Information"]) {
+        const text = entry.extraData["Additional Information"];
+        if (
+            //"resident" " live" " work" "eligible populations in"
+            text
+                .toLowerCase()
+                .match(/(resident|\slive|\swork|eligible\spopulations\sin)/)
```

Comment:
I suspect this will resulting in us flagging more than we need (e.g. `Berkshire Community College Field House` has additional info "FOR THOSE 65 AND ABOVE, OR WHO HAVE TWO OR MORE MEDICAL CONDITIONS ELIGIBLE UNDER PHASE 2. ID will be required." This will be flagged as "may be restricted" even though it is no more restrictive than any other place.

Reply:
actually - it won't be flagged (and isn't) b/c it only has "eligible" and not "eligible populations in"

Reply:
Yeah that's why I did "eligible populations in" - that way it is connoting where you live as opposed to the fact that you're eligible due to phases